src/test/java/com/google/devtools/build/lib/remote/DiskCacheIntegrationTest.java

```diff
@@ -212,6 +212,27 @@ public void remoteExecWithDiskCache_hitDiskCache() throws Exception {
     events.assertContainsInfo("2 disk cache hit");
   }
 
+  @Test
+  public void remoteExecWithDiskCache_resetDiskCache() throws Exception {
+    // Download all outputs to populate the disk cache.
+    doRemoteExecWithDiskCache("--remote_download_all");
+
+    // Assert: Should hit the disk cache
+    events.assertContainsInfo("2 disk cache hit");
+
+    // Delete the cache directory entirely.
+    getWorkspace().getFileSystem().getPath(getDiskCacheDir()).deleteTree();
+
+    // Simulates a server restart
+    createRuntimeWrapper();
```

Comment:
nit: using `cleanAndRestartServer` is more readable.

Reply:
PTAL